pyflakes/checker.py

```diff
@@ -873,6 +873,28 @@ def getDocstring(self, node):
 
         return (node.s, doctest_lineno)
 
+    def handleTypeComments(self, node):
+        def extractTypeComments(typeComment):
+            if '->' in typeComment:
+                for token in node.type_comment.split(' -> '):
+                    if token[0] == '(' and token[-1] == ')':
+                        token = token[1:-1]
+                    if ',' in token:
+                        for part in token.split(','):
+                            yield part.strip()
+                    else:
+                        yield token
+            else:
+                yield typeComment
+
+        typeComment = getattr(node, 'type_comment', None)
+        if typeComment is not None:
+            for comment in extractTypeComments(typeComment):
+                typeNode = ast.Name(comment, ast.Load())
```

Comment:
I realize that this is likely not the best way to create a node, but it worked. Happy for suggestions on how to do this better.